ThreeBotPackages/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
The same as https://github.com/threefoldtech/jumpscaleX_threebot/pull/393.
Why do we have yarn.lock here?